Sources_v3/ChatClient.swift

```diff
@@ -166,6 +166,7 @@ public class Client<ExtraData: ExtraDataTypes> {
         // All production workers
         let workerBuilders: [WorkerBuilder] = [
             MessageSender<ExtraData>.init,
+            MessageEditor<ExtraData>.init,
```

Comment:
I don't know, the name doesn't work for me, but I don't have any other idea than `MessageChangesUpdater` and I don't like it either.